src/dynamic_methods/radius_expansion_method_with_servo_control.py

```diff
@@ -61,7 +61,7 @@ def Run(self, parameters, ini_path):
         self.Initialization(parameters, ini_path)
         packing_num = self.parameters["packing_num"]
         target_packing_density = self.parameters["random_particle_generation_parameters"]["target_packing_density"]
-        target_packing_density_list = [target_packing_density-0.02, target_packing_density-0.05, target_packing_density-0.075, target_packing_density-0.1, target_packing_density-0.15]
+        target_packing_density_list = [target_packing_density-0.002, target_packing_density-0.05, target_packing_density-0.075, target_packing_density-0.1, target_packing_density-0.15]
```

Comment:
Maybe I should make this list as an input

Reply:
Ok. If you find that using "target_packing_density-0.002" can speed up the simulation, maybe we can expand the list as [target_packing_density-0.002, target_packing_density-0.02, target_packing_density-0.05, target_packing_density-0.075, target_packing_density-0.1, target_packing_density-0.15]. What do you think?

Reply:
It is ok

Reply:
ok